### PR TITLE
MD rules, and change other stuff so its ✅

### PR DESCRIPTION
**Pull request by xamionex**
Replace this line with a short description of your change.
I won't. You're a good line :)

Anyway, people should know MD rules tbh, use [Markdownlint](https://marketplace.visualstudio.com/items?itemName=DavidAnson.vscode-markdownlint) if you have VSCode
_also faky doesn't know how 2 github B)_

- Apparently Alternative fix to LSX works _sometimes_
  - But I'm keeping it in if it helps anyone.
- And also Origin accepts - and + but I didn't know since It didn't allow me, turns out it doesn't accept numpad input \~w~

### CODE REVIEW

**Comment by xamionex** [comment]
[as per usual my gitbook is here](https://amione.gitbook.io/northstarwiki/) _if you dont see changes just CTRL + F5_

File: docs/installing-northstar/troubleshooting.md [hunk 1]
```diff
@@ -146,7 +151,7 @@ The real error is most likely slightly higher. Please post that in issues or the
 
 ## I can't play the Campaign <a href="#campaign" id="campaign"></a>
 
-Campaign is not supported in 1.4, but you could do one of these **"**_**fixes**_**"**
+Campaign is not supported in Northstar, but you could do one of these "**_fixes_**"
```
Comment (GeckoEidechse) [inline docs/installing-northstar/troubleshooting.md]:
```suggestion
Campaign is not supported in Northstar yet, but you could do one of these "**_fixes_**"
```

Support will be added eventually :D

**Review by GeckoEidechse — COMMENTED** [review]
Otherwise looks good ^^

**Comment by GeckoEidechse** [comment]
Aight, looks good <3

**Pull request merged**